src/firebase.js

```diff
@@ -0,0 +1,15 @@
+import firebase from "firebase"
+
+const config = {
+    apiKey: "AIzaSyAmCn-kpDErfZ4M5iOysQCH81cuvivwvFY",
+    authDomain: "epicure-firestore.firebaseapp.com",
+    databaseURL: "https://epicure-firestore-default-rtdb.europe-west1.firebasedatabase.app",
+    projectId: "epicure-firestore",
+    storageBucket: "epicure-firestore.appspot.com",
+    messagingSenderId: "487415923981",
+    appId: "1:487415923981:web:1a191a8602e2541eaf9c44"
+  };
+  // Initialize Firebase
+  firebase.initializeApp(config);
+  
```

Comment:
Nice job connecting to firebase, a great next step would be to create a specific service to handle everything firebase related for ease of development, without the need to reference and use each call to the db separately

Reply:
This might also be a great place to initialize firestore and get the necessary refs

Reply:
Thanks! what type of service are you referring to ?

Reply:
A .ts file that houses and exports all of your firestore manipulation logic, this can be a class with methods or a bunch of exported functions

Reply:
I'll try it out